### ISSUE TITLE
Scotch box doesn't sync back after host upgraded to Sierra

### ISSUE THREAD

**Issue by maykino** [scotch-io/scotch-box#268]
Hi, it seems like after the last update to OSX Sierra my scotchbox stopped syncing back for my Symfony application. I did some troubleshooting and it seems like it is only affecting my Symfony project with a custom Vagrantfile setup. We also confirmed the same issue on other machines. Simply when you change a file on the client machine (when you are ssh-ed to scotchbox), or even run composer install/update you don't see any changes in any files on your host machine. Here is my Vagrantfile:

Vagrant.configure("2") do |config|

```
config.vm.provision "fix-no-tty", type: "shell" do |s|
    s.privileged = false
    s.inline = "sudo sed -i '/tty/!s/mesg n/tty -s \\&\\& mesg n/' /root/.profile"
end

config.vm.box = "scotch/box"
config.vm.provider :virtualbox do |vb|
    vb.customize ["modifyvm", :id, "--memory", "4096"]
    vb.customize ["modifyvm", :id, "--cpus", "4"]   
end

config.vm.network "private_network", ip: "192.168.33.3"
config.vm.hostname = "scotchbox12"
config.vm.synced_folder ".", "/var/www", type: "rsync",
    rsync__exclude: ["app/", "bin/", "features/", "src/", "tests/", "vendor/", "web/"]

config.vm.synced_folder "app/", "/var/www/app", type: "nfs"
config.vm.synced_folder "bin/", "/var/www/bin", type: "nfs"
config.vm.synced_folder "features/", "/var/www/features", type: "nfs"
config.vm.synced_folder "src/", "/var/www/src", type: "nfs"
config.vm.synced_folder "tests/", "/var/www/tests", type: "nfs"
config.vm.synced_folder "vendor/", "/var/www/vendor", type: "nfs"
config.vm.synced_folder "web/", "/var/www/web", type: "nfs"

# Change the document root folder
config.vm.provision "shell", privileged: false, inline: <<-SHELL
    sudo ln -s /home/vagrant/.rbenv/shims/ruby /usr/bin/ruby
    sudo /home/vagrant/.rbenv/shims/gem install compass
    sudo ln -s /home/vagrant/.rbenv/shims/compass /usr/bin/compass
    sudo ln -s /home/vagrant/.rbenv/shims/sass /usr/bin/sass
    sudo sed -i s,/var/www/public,/var/www/web,g /etc/apache2/sites-available/000-default.conf
    sudo sed -i s,/var/www/public,/var/www/web,g /etc/apache2/sites-available/scotchbox.local.conf
    sudo service apache2 restart
SHELL
```

end

**Comment by maykino** [comment]
Now I changed the "synced_folder" line to 

`config.vm.synced_folder ".", "/var/www", type: 'nfs', mount_options: ['rw', 'vers=3', 'tcp', 'fsc', 'actimeo=2'], rsync__exclude: ["app/", "bin/", "src/", "tests/", "vendor/", "web/"]`
    
and it seems like syncing started working but I ran into another issues. Whenever I try to run composer update I get the following error on every package:

<img width="972" alt="screen shot 2016-10-26 at 5 14 35 pm" src="https://cloud.githubusercontent.com/assets/4440410/19749681/baf7994a-9b9f-11e6-91cc-3b7237294faa.png">

It seems like something is messed up with file/folder permissions? When I remove the whole vendor folder and subsequently run vagrant halt/destroy/up and then composer install from the client machine it works fine. However any further composer update messes up everything and simply removes packages one by one.
